packages/web-react/src/components/Footer/README.md

```diff
@@ -0,0 +1,255 @@
+# Footer
+
+The Footer is a highly variable composition. It comes in several ready-to-use design variants, but can be customized to
+achieve your specific design goals.
+
+## Composition
+
+The Footer supports several ready-to-use building blocks:
+
+- [Navigation Links](#navigation-links)
+- [Product Logo](#product-logo)
+- [Social Media Links](#social-media-links)
+- [Language Switch](#language-switch)
+- [Secondary Links](#secondary-links)
+
+This is how all supported building blocks of the Footer build up the complete composition:
+
+```jsx
+<Footer UNSAFE_className="bg-cover pt-1100 pb-1000">
+  <Container>
+    {/* Grid with navigation links */}
+    {/* Grid with product logo, social media links and language switch */}
+    {/* Flex with secondary links */}
+  </Container>
+</Footer>
+```
+
+👉 Good to know:
+
+- To achieve the desired design, you can use the provided building blocks in any order you need.
+- All building blocks are optional, so you can use only the ones you need.
+- You are not limited just to the provided building blocks. You can add your own content or building blocks (such as a
+  newsletter subscription form) as needed.
+- Use [Grid][grid] and [Flex][flex] components to create the desired layout.
+- Use [Stack][stack] or [Divider][divider] components to organize individual sections.
+- Use spacing utility classes like `mb-*` or `pt-*` to achieve desired spacings between components.
```

Comment:
we should mention spacingprops as well

Reply:
added mention about spacing props and link to readme